platform/pensando/sonic-platform-modules-dpu/dpu/service/dpu-db-util.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Pensando DPU health status provider and chassis state db util service
+Requires=system-health.service
+After=system-health.service
+
+[Service]
+ExecStartPre=/bin/sleep 75
```

Comment:
Why the arbitrary sleeps?

Reply:
Removed it as system-health.service dependency gone.